src/Microsoft.DotNet.Tools.Pack/Program.cs

```diff
@@ -24,29 +23,30 @@ public static int Main(string[] args)
             app.Description = "Packager for the .NET Platform";
             app.HelpOption("-h|--help");
 
+            var input = app.Option("-i|--input <INPUT_DIR>", "Directory from where the assets to be packages are going to be picked up", CommandOptionType.SingleValue);
```

Comment:
This is how it was before, for output and project at least, I don't want to change it.